pkgs/development/libraries/science/math/magma/default.nix

```diff
@@ -19,7 +19,24 @@ in stdenv.mkDerivation {
 
   enableParallelBuilding=true;
 
-  # MAGMA's default CMake setup does not care about installation. So we copy files directly.
+  # we will not build tests as erroring on compilation
```

Comment:
Why does this copy a part of the stdenv?

Reply:
We need to run “make magma” and then run make for sparse. Let me know if I missed a more elegant way to do this? I don’t believe the makeflags options are sufficient as we need to run two separate make commands

Reply:
You can pass `buildFlags = [ "magma" "magma_sparse" ];`